### DISCUSSION TITLE
Security - How to satisfy internal company security to be allowed use DBADash?

### DISCUSSION THREAD

**Discussion by HansLindgren** [trimble-oss/dba-dash#598] [Q&A] [answered]
I am working for a company that is considering implementing DBADash, as several of my DBA colleagues have had positive experiences with the software. I have also personally evaluated it in a sandbox environment.

Before the company can proceed with DBADash, it must undergo an internal security review. I am seeking suggestions on key aspects to emphasize and the best strategies to improve the likelihood of DBADash passing this assessment.

I would greatly appreciate any input or advice that you might be willing to share. Thank you!

**Comment by DavidWiseman** [comment, marked as answer]
DBA Dash is genuinely free software, provided under the [MIT License](https://dbadash.com/docs/help/security/) agreement.  There is no data harvesting - all the data collected goes to the central repository database that **you control**.  You also have access to the source control so you can see how it works under the hood.   DBA Dash was developed by me and is owned by [Trimble](https://en.wikipedia.org/wiki/Trimble_Inc.)

The [security](https://dbadash.com/docs/help/security/) section of the documentation might be worth a read.  It's possible to configure DBA Dash to run with restricted access and the document addresses some other security concerns.  

If you have any specific questions, feel free to post back and I'll answer them if I can.
